README.md

```diff
@@ -1,21 +1,23 @@
 # Social Diversity for Children Foundation
 
-Program registration platform for [Social Diversity for Children Foundation](https://www.socialdiversity.org).
+Program registration platform for [Social Diversity for Children Foundation](https://www.socialdiversity.org). 
+
+Currently in development!
 
 This project was generated with [superplate](https://github.com/pankod/superplate).
```

Comment:
do we need this?

Reply:
Don't think it matters or is bad to have